com.gitlab.KRypt0n_.an-anime-game-launcher.yml

```diff
@@ -0,0 +1,109 @@
+app-id: com.gitlab.KRypt0n_.an-anime-game-launcher
+runtime: org.gnome.Platform
+runtime-version: "41"
+sdk: org.gnome.Sdk
+separate-locales: false
+command: com.gitlab.KRypt0n_.an-anime-game-launcher
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11 # Not fallback-x11, because while the launcher supports Wayland, Wine doesn't (yet)
+  - --socket=wayland
```

Comment:
```suggestion
```

Reply:
The launcher supports Wayland, it's only once the game is launched that X11 is needed